copier/main.py

```diff
@@ -810,6 +711,33 @@ def _git_initialize_repo(self):
         git("config", "--unset", "user.name")
         git("config", "--unset", "user.email")
 
+    @cached_property
+    def _default_renderer(self) -> Renderer:
+        return Renderer(
+            template=self.template,
+            subproject=self.subproject,
+            _render_allowed=self._render_allowed,
+            pretend=self.pretend,
+            jinja_env=self.jinja_env,
+            answers_relpath=self.answers_relpath,
+            _render_context=self._render_context(),
+            _items_keys=self.project_mode_keys,
+        )
+
+    def _render_string(self, string: str) -> str:
+        return self._default_renderer.render_string(string)
+
+    @cached_property
+    def project_mode_keys(self) -> Iterable[str]:
+        raw_items = self.template.config_data.get("items")
+        if not raw_items:
+            return ()
+        return (
+            [raw_items]
+            if isinstance(raw_items, str)
+            else cast(collections.abc.Iterable, raw_items)
```

Comment:
If it is a string, then it should probably get rendered using the native jinja env, to allow for complex scenarios. Right?